docs/zh_CN/2023-02-14-so-whats-next.md

```diff
@@ -582,7 +582,7 @@ core-js的自动polyfill/优化不应该仅在Babel中可用。虽然core-js项
 
 另一个更常见的问题是来自全局（global）和纯（pure）core-js版本的polyfill的重复。纯core-js / babel-runtime旨在用于库的代码中，因此如果你使用全局版本的core-js而且你的依赖项也加载了一些不带全局命名空间污染的core-js副本，那么这是一种正常情况。它们使用不同的内部结构，但它们之间共享类似的代码是有问题的。
 
-我正考虑在transpiler或bundler插件中解决这个问题，与前一个类似（但当然，更复杂一点）——我们可以将纯版本的导入替换为全局版本导入，并删除目标引擎中不必要的polyfill。
+我正考虑在转译器或捆绑器插件中解决这个问题，与前一个类似（但当然，更复杂一点）——我们可以将纯版本的导入替换为全局版本导入，并删除目标引擎中不必要的polyfill。
```

Comment:
```suggestion
我正考虑在转译器或捆绑器插件中解决这个问题，与前一个类似（但当然，更复杂一点）——我们可以将纯版本的导入替换为全局版本导入，并删除目标引擎中不必要的 polyfill。
```

Reply:
When Chinese and English words are in the same sentence, there should be a space between the Chinese and English words.